documentation/meta/monitoring/cloudwatch_logs/index.md

```diff
@@ -0,0 +1,246 @@
+EC2 instance. In both cases, the log streams include the identifier for the task
+or instance. For ECS, the streams have the following pattern:
+
+```
+ecs/<container name>/<task id>
+```
+
+For EC2, the log stream name is simply the instance identifier. Notably, the
+ingestion server has separate log groups for the workers and the coordinator.
+
+To query logs for a specific ECS task or EC2 instance, query its corresponding
+log stream.
+
+To query the logs for an entire application over a given time period, ignore the
+task IDs entirely, and simply query the full log group. This is especially
+useful if you know that an event happened within a particular period of time but
+do not know the task or EC2 instance identifier and therefore which specific log
+stream to query. Because the ingestion server splits its logs in two separate
+groups, **it is not possible to query all logs for the ingestion server in a
+single query.** You will need to separately query both the ingestion server and
+work log groups for the environemnt.
+
+To query all logs for a log group, go to the log group page and click the orange
+"Search log group" button in the upper right hand corner.
+
+![Search log group button in CloudWatch](/meta/monitoring/cloudwatch_logs/search_log_group.png)
+
+It is not possible to query specific ECS container log streams via the log
+groups UI. For example, if you want to query only the Django container logs for
+the API log group, you will not be able to narrow your query to exclude the
+nginx log streams explicitly. Instead, structure your query such that it will
+only match log events from the Django application (or vice-versa).
+
+### Logs Insights
+
+Logs Insights is very powerful. It is particular useful for deriving numerical
+and statistical data from both structured and unstructured logs, though the
+latter require additional parsing. Logs Insights has its own SQL-like query
+syntax,
+[the documentation for which](https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/CWL_QuerySyntax.html)
+should be the read before attempting to use Logs Insights for anything serious.
+
+The Logs Insights tutorials present sufficent examples for different types of
+queries that you may need to do. A good tip is to think of the Logs Insights
+query as a multi-step extract, transform, and load/aggregate tool. First,
+extract the specific log events that concern you. If you're querying to find the
+average ES query time logged in a particular format, first narrow the log
+streams down to just the service sending those logs. Next, use `parse` to
+extract the relevant information from the unstructured logs. Finally, use
+`stats` to aggregate the data and find the statistical information relevant to
+you. The following are two simple examples that can serve as starting points for
+querying structured and unstructured log data in Logs Insights for our
+application. Please note that logs insights queries are **not free**.
```

Comment:
Should we use the Sphinx `admonition` to make this more visible?

Reply:
Yup! Done in the next commit.